iml-wire-types/src/db.rs

```diff
@@ -602,7 +607,7 @@ impl Name for LnetConfigurationRecord {
 #[derive(
     Debug, serde::Serialize, serde::Deserialize, Eq, PartialEq, Ord, PartialOrd, Clone, Hash,
 )]
-pub struct DeviceId(String);
+pub struct DeviceId(pub String);
```

Comment:
This does not need to be `pub`:

```suggestion
pub struct DeviceId(String);
```

Reply:
It needs to be `pub` because we're constructing `DeviceId` in tests - see `flat_devices.rs`, `breadth_first_iterator.rs`, `virtual_device.rs`.